e2e/subscriptions-cancellation/subscriptions-cancellation.e2e.ts

```diff
@@ -37,6 +37,13 @@ it('should subscribe and cancel', async () => {
 
   await setTimeout(1_000); // allow some calmdown time (TODO: avoid magic numbers, any other approach to this?)
 
-  expect(gw.getStd('out').replaceAll(' ', '')).toContain('ITERABLE>NEXT>END'); // TODO: there shouldnt be multiple "END"s
```

Comment:
Please put back this TODO. The `onEnd` gets called many times instead of just once.

Reply:
Done ✅